plugins/modules/net_tools/nmcli.py

```diff
@@ -168,6 +209,43 @@
         type: list
         elements: str
         version_added: 4.4.0
+    routes6_extended:
+        description:
+            - The list of IPv6 routes but with parameters.
+        type: list
+        elements: dict
+        suboptions:
+            ip:
+                description:
+                    - IP or prefix of route.
+                    - Use the format C(fd12:3456:789a:1::/64).
+                type: str
+                required: true
+            next_hop:
+                description:
+                    - Use the format C(2001:dead:beef::1).
+                type: str
+            metric:
+                description:
+                    - Route metric.
+                type: int
+            table:
+                description:
+                    - The table to add this route to.
+                    - The default depends on C(ipv4.route-table).
```

Comment:
Shouldn't this probably be
```suggestion
                    - The default depends on C(ipv6.route-table).
```
?